tests/test_component.py

```diff
@@ -1,29 +1,21 @@
-from nexus_constructor.component import ComponentModel, DependencyError
-from nexus_constructor.nexus.nexus_wrapper import NexusWrapper
+from nexus_constructor.component import DependencyError
 from cmath import isclose
-from typing import Any
 from PySide2.QtGui import QVector3D
 import pytest
-from uuid import uuid1
-
-
-def _add_component_to_file(
-    nexus_wrapper: NexusWrapper,
-    field_name: str,
-    field_value: Any,
-    component_name: str = "test_component",
-):
-    component_group = nexus_wrapper.nexus_file.create_group(component_name)
-    component_group.create_dataset(field_name, data=field_value)
-    return component_group
+from pytest import approx
+from .helpers import create_nexus_wrapper, add_component_to_file
+from nexus_constructor.geometry import (
+    CylindricalGeometry,
+    OFFGeometryNexus,
+    OFFGeometryNoNexus,
+)
 
 
 def test_can_create_and_read_from_field_in_component():
     field_name = "some_field"
     field_value = 42
-    nexus_wrapper = NexusWrapper(str(uuid1()))
-    component_group = _add_component_to_file(nexus_wrapper, field_name, field_value)
-    component = ComponentModel(nexus_wrapper, component_group)
+    nexus_wrapper = create_nexus_wrapper()
```

Comment:
If we're creating this wrapper and component for every test does it make sense just to do it in a fixture?

Reply:
I did consider it, but in this case I feel the setup is relevant to the tests rather than just being required boilerplate, so I'm not too keen on "hiding it" in a fixture. I opted instead for the helper functions, which reduce it to a couple of lines whilst still making to clear what the scenario for the test is.

Reply:
Sure, that's reasonable.